AlfrescoApp.xcodeproj/project.pbxproj

```diff
@@ -5416,6 +5515,16 @@
 				GCC_WARN_UNUSED_VARIABLE = YES;
 				IPHONEOS_DEPLOYMENT_TARGET = 8.4;
 				ONLY_ACTIVE_ARCH = YES;
+				OTHER_CFLAGS = (
+					"-DCLOUD_OAUTH_KEY=\"@\\\"${ALFRESCO_IOS_CLOUD_OAUTH_KEY}\\\"\"",
+					"-DCLOUD_OAUTH_SECRET=\"@\\\"${ALFRESCO_IOS_CLOUD_OAUTH_SECRET}\\\"\"",
+					"-DINTERNAL_CLOUD_API_KEY=\"@\\\"${INTERNAL_CLOUD_API_KEY}\\\"\"",
+					"-DFLURRY_API_KEY=\"@\\\"${ALFRESCO_IOS_FLURRY_API_KEY}\\\"\"",
+					"-DQUICKOFFICE_PARTNER_KEY=\"@\\\"${QUICKOFFICE_PARTNER_KEY}\\\"\"",
+					"-DHOCKEYAPP_APPID=\"@\\\"${ALFRESCO_IOS_HOCKEYAPP_APPID}\\\"\"",
+					"-DHOCKEYAPP_UPDATES=\"@\\\"${HOCKEYAPP_UPDATES}\\\"\"",
+					"-DGA_API_KEY=\"@\\\"${ALFRESCO_IOS_GA_API_KEY}\\\"\"",
+				);
```

Comment:
I'm not sure why these seem to have moved from the `xcconfig` file to the project file?